diagnostics-with-elk/azuredeploy.json

```diff
@@ -123,17 +123,12 @@
             "type": "string",
             "defaultValue": "14.04.2-LTS",
             "allowedValues": [
-                "12.04.2-LTS",
-                "12.04.3-LTS",
-                "12.04.4-LTS",
-                "12.04.5-LTS",
                 "12.10",
```

Comment:
Portal should be good indicator. Other way is to get through PS or CLI. 15.10 will probably get removed soon since its EOL from Canonical